src/supernode.cpp

```diff
@@ -160,19 +162,19 @@ void Supernode::setHttpRouters(ConnectionManager& httpcm)
     // httpcm.addRouter(dapi_router);
 
     // Router http_router;
```

Comment:
Why not "using namespace graft::supernode::request;" here instead of "globally" ?

Reply:
Which line of code it does relate to?

Reply:
Got it
My idea is to put content of some namespace inside of some limited area. Not in topmost. The higher - the worse.
What you suggestion would give us and why it's better than this?
Could you make it clear?